src/make_app.ts

```diff
@@ -65,10 +70,17 @@ export function makeApp(options: AppOptions): PolyserveApplication {
 
     if (headers) {
       for (const header in headers) {
-        (<any>res).append(header, headers[header]);
+        (<any>res).setHeader(header, headers[header]);
       }
     }
-    send(req, filePath).pipe(res);
+
+    const _send = send(req, filePath, {
+      lastModified: false,
+      cacheControl: false,
+    });
+    // Uncomment this to disable disable 304s from send() and always compile
+    // _send.isFresh = () => false;
```

Comment:
What's the issue here? How does this relate to the lastModified and cacheControl options just above?

Also, nit, `disable disable` -> `disable`